From 9751f057f61da6507dcff83857247997f5c7791e Mon Sep 17 00:00:00 2001
From: BBaoVanC <bbaovanc@bbaovanc.com>
Date: Sun, 29 May 2022 16:51:01 -0500
Subject: [PATCH 1/2] Remove GPG signing from gitconfig

---
 .gitconfig | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/.gitconfig b/.gitconfig
index d71bd4d..31bf49c 100644
--- a/.gitconfig
+++ b/.gitconfig
@@ -1,11 +1,8 @@
 [user]
 	name = BBaoVanC
 	email = bbaovanc@bbaovanc.com
-	signingKey = CB7CCED9209E47B9B99B8E1D18089E4E3CCF1D3A
 [core]
 	editor = nvim
-[commit]
-	gpgsign = true
 [filter "lfs"]
 	process = git-lfs filter-process
 	required = true
@@ -28,5 +25,3 @@
 	insteadOf = https://git.bbaovanc.com/
 [url "ssh://git@git.boba.best/"]
 	insteadOf = https://git.boba.best/
-[tag]
-	gpgSign = true
-- 
2.42.0

