From 4619971fe0f1d370cb26f2aee8a86e5f7bad3105 Mon Sep 17 00:00:00 2001
From: BBaoVanC <bbaovanc@bbaovanc.com>
Date: Tue, 22 Mar 2022 17:03:04 -0500
Subject: [PATCH 2/3] Don't GPG sign commits

---
 .gitconfig | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/.gitconfig b/.gitconfig
index a266cdd..26f25ec 100644
--- a/.gitconfig
+++ b/.gitconfig
@@ -1,11 +1,8 @@
 [user]
 	name = BBaoVanC
 	email = bbaovanc@bbaovanc.com
-	signingKey = CB7CCED9209E47B9B99B8E1D18089E4E3CCF1D3A
 [core]
 	editor = nvim
-[commit]
-	gpgsign = true
 [filter "lfs"]
 	process = git-lfs filter-process
 	required = true
@@ -23,5 +20,3 @@
 	browser = firefox-developer-edition
 [browser "firefox-developer-edition"]
 	cmd = firefox-developer-edition
-[tag]
-	gpgSign = true
-- 
2.35.1

