From fc1a1898622c7deb57c09d68cc67f4afd214af64 Mon Sep 17 00:00:00 2001
From: BBaoVanC <bbaovanc@bbaovanc.com>
Date: Sun, 29 May 2022 16:51:31 -0500
Subject: [PATCH 2/2] Remove SSH cloning from gitconfig

---
 .gitconfig | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/.gitconfig b/.gitconfig
index 31bf49c..a15b25f 100644
--- a/.gitconfig
+++ b/.gitconfig
@@ -18,10 +18,3 @@
 	fork-point = !zsh -c 'diff --old-line-format='' --new-line-format='' <(git rev-list --first-parent "${1:-master}") <(git rev-list --first-parent "${2:-HEAD}") | head -1' -
 [pull]
 	ff = only
-
-[url "ssh://git@github.com/"]
-	insteadOf = https://github.com/
-[url "ssh://git@git.bbaovanc.com/"]
-	insteadOf = https://git.bbaovanc.com/
-[url "ssh://git@git.boba.best/"]
-	insteadOf = https://git.boba.best/
-- 
2.42.0

