From b5f0c93edd497162ef54ba26685d3c9d99d4fe2c Mon Sep 17 00:00:00 2001
From: BBaoVanC <bbaovanc@bbaovanc.com>
Date: Sun, 29 May 2022 16:51:31 -0500
Subject: [PATCH 2/2] Remove SSH cloning from gitconfig

---
 .gitconfig | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/.gitconfig b/.gitconfig
index efc6d92..f222aa7 100644
--- a/.gitconfig
+++ b/.gitconfig
@@ -16,10 +16,3 @@
 	isolog = log --date=iso-strict
 [pull]
 	ff = only
-
-[url "ssh://git@github.com/"]
-	insteadOf = https://github.com/
-[url "ssh://git@git.bbaovanc.com/"]
-	insteadOf = https://git.bbaovanc.com/
-[url "ssh://git@git.boba.best/"]
-	insteadOf = https://git.boba.best/
-- 
2.36.1

